### ISSUE TITLE
Stable release

### ISSUE THREAD

**Issue by diegogoding** [mayn/packerlicious#108]
### discussion
This looks like a nice little package, one major blocker for me using this is the statement below in the [readme](https://github.com/mayn/packerlicious/blob/master/README.rst) statement: 
```
Project follows semantic versioning , v0.x.x API should be considered unstable, API will change frequently, please plan accordingly.
```
What's the exceptions and tasks left to have a stable version?

I appreciate that you put in the effort and thought to make declaration about the expected instability of the API upfront.

**Comment by mayn** [comment]
Hi @diegogoding. Nothing at this point. 

I’ll leave this discussion open for a bit(few weeks), see if there is any additional commentary, then take the leap to release as “stable”

TLDR:
The original disclaimer was to allow for rapid development changes during project inception without the fear of breaking anyone investing time working with the project. 

Python 2/3 compatibility(done), receiving user feedback and not preemptively declaring “stable” were the main tasks outstanding.

**Comment by mayn** [comment]
v1.0.0 has been release, this issue will be closed
